cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -118,8 +123,7 @@ func (c *CStorVolumeController) cStorVolumeEventHandler(
 		if err != nil {
 			return common.CVStatusError, err
 		}
-
-		return common.CVStatusIgnore, nil
+		return common.CVStatusInit, nil
```

Comment:
this change sets the status on CV CR.. is that intended behavior?

Reply:
ok.. this state change doesn't seem to impact anything wrt state mechanism.. but, the current status of CV CR is lost, which as well seems to be fine in the case of Add operation

Reply:
> this change sets the status on CV CR.. is that intended behavior?
Yes